### PR TITLE
Allow nexthop type of None in panos_static_route module

### PR DESCRIPTION
**Pull request by freakinhippie**
This PR allows setting a nexthop_type of null in the yaml, which is translated to an empty string, and then interpreted as None.

### CODE REVIEW

**Comment by freakinhippie** [comment]
I'm working on an update to change it to `'none'` rather than `null`. I just need to run a few tests to validate it works properly.

**Comment by freakinhippie** [comment]
@shinmog I've completed your requested changes.

File: docs/modules/panos_static_route_module.md [hunk 1]
```diff
@@ -80,6 +80,16 @@ Create static routes on PAN-OS devices.
...
 
+    - name: Create route 'Test-Five'
+      panos_static_route:
+        ip_address: '{{ fw_ip_address }}'
+        username: '{{ fw_username }}'
+        password: '{{ fw_password }}'
+        name: 'Test-Five'
+        destination: '5.5.5.0/24'
+        # use 'none' for a nexthop type of 'None'
+        nexthop_type: none
```
Comment (shinmog) [inline docs/modules/panos_static_route_module.md]: `'none'`

File: examples/fw_static_route_add.yml [hunk 2]
```diff
@@ -55,3 +55,13 @@
...
+
+  - name: Create route 'Test-Five'
+    panos_static_route:
+      ip_address: '{{ fw_ip_address }}'
+      username: '{{ fw_username }}'
+      password: '{{ fw_password }}'
+      name: 'Test-Five'
+      destination: '5.5.5.0/24'
+      # use null for a nexthop type of 'None'
+      nexthop_type: null
```
Comment (shinmog) [inline examples/fw_static_route_add.yml]: Probably ok to remove the comment, but the setting should be `'none'` now that you changed this.

File: library/panos_static_route.py [hunk 3]
```diff
@@ -128,6 +128,15 @@
...
     virtual_router: 'VR-Two'
+
+- name: Create route 'Test-Five'
+    panos_static_route:
+    ip_address: '{{ fw_ip_address }}'
+    username: '{{ fw_username }}'
+    password: '{{ fw_password }}'
+    name: 'Test-Five'
+    destination: '5.5.5.0/24'
+    nexthop_type: none
```
Comment (shinmog) [inline library/panos_static_route.py]: `'none'`

Comment (freakinhippie) [reply library/panos_static_route.py]: @shinmog I've made the requested changes. Thanks.

**Pull request merged**